emcc/release.sh

```diff
@@ -9,7 +9,7 @@ tar -xjf $OS-amd64-github-release.tar.bz2
 
 ./bin/$OS/amd64/github-release release -u apiaryio -r drafter --tag $TAG
```

Comment:
Do we need to check for `pre` in TAG and mark the github release as pre release?